test/integration/test_mail.py

```diff
@@ -0,0 +1,178 @@
+# Copyright 2015-2025 Earth Sciences Department, BSC-CNS 
+# 
+# This file is part of Autosubmit. 
+# 
+# Autosubmit is free software: you can redistribute it and/or modify 
+# it under the terms of the GNU General Public License as published by 
+# the Free Software Foundation, either version 3 of the License, or 
+# (at your option) any later version. 
+# 
+# Autosubmit is distributed in the hope that it will be useful, 
+# but WITHOUT ANY WARRANTY; without even the implied warranty of 
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the 
+# GNU General Public License for more details. 
+# 
+# You should have received a copy of the GNU General Public License 
+# along with Autosubmit.  If not, see <http://www.gnu.org/licenses/>. 
+
+from random import randint, sample
+from typing import Tuple, cast
```

Comment:
`cast` is now also not used,

![image](https://github.com/user-attachments/assets/495d0f96-3af5-4033-8c72-e654ebe61cc2)